website/content/ja/contribute/projects/translations/index.md

```diff
@@ -0,0 +1,26 @@
+---
+title: 翻訳
```

Comment:
https://cloud-native.slack.com/archives/CL3SL0CP5/p1717604563468949

Reply:
I agree with using localization over translation. We can update this once the original document is updated.

Reply:
Great idea!!

Like you, I think this improvement is outside the scope of this PR. We shouldn't block the merging of this PR because of this topic. Instead, let's create a new PR for it, and make the updates from the English content.